GridLock/GridLockViewController.swift

```diff
@@ -10,10 +10,67 @@ import UIKit
 
 class GridLockViewController: UIViewController {
     
+    var lines: [UIView] = []
+    let lineColor: UIColor
+    let lineWidth: CGFloat
+    
+    init(lineColor: UIColor = .blue, lineWidth: CGFloat = 5.0) {
+        self.lineColor = lineColor
+        self.lineWidth = lineWidth
+        
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
```

Comment:
Perhaps just call `super` and log a warning here instead of the fatal error?

Reply:
If we don't throw a fatalError here we get errors because we haven't initialised the `let` constants at the top of the class